src/inputValidator.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const digits = '0123456789';
+
+const isValidInput = (input) => {
+  const hasExactlyFourDigits = input.length === 4;
```

Comment:
When you use some static values like 4 here especially when you use it in two or more places, it's better to move it to some variable.